torch/testing/_internal/common_utils.py

```diff
@@ -255,7 +255,8 @@ def chunk_list(lst, nchunks):
 
 # sanitize filename e.g., distributed/pipeline/sync/skip/test_api.py -> distributed.pipeline.sync.skip.test_api
 def sanitize_test_filename(filename):
-    strip_py = re.sub(r'.py$', '', filename)
+    strip_py = re.sub(r'.*/pytorch/test/', '', filename)
```

Comment:
```suggestion
    # Replace initial dot with underscore
    strip_py = re.sub(r'^\.', '_', filename)
```

Reply:
this wouldn't work because the sanitized test file path needs to be one of those listed in 
https://github.com/pytorch/pytorch/blob/1fc3e1e1fbcd70a134560b420bec932905156617/test/run_test.py#L31